dashboard/test/ui/features/star_labs/applab/shared_apps.feature

```diff
@@ -79,6 +79,7 @@ Feature: App Lab Scenarios
     Then element "#checkbox1" is checked
     And element "#checkbox2" is checked
 
+  @no_ie
```

Comment:
Is this still needed with the below fix that adds a wait?

Reply:
The comment above the step definition is that 'IE does not register the key presses in this step`. So does adding a wait resolve this issue? Seems like if we're on IE, we would just wait 30 sec and then proceed to next step.

Also, I'm new to UI tests, but do we check on IE still? I don't see IE listed at https://test-studio.code.org/ui_test/test_status_UI.html.

I also see you added mentioned `@no_ie` tags in [this past PR](https://github.com/code-dot-org/code-dot-org/pull/27789/files).

Either way, I'm happy to remove tag!

Reply:
> Also, I'm new to UI tests, but do we check on IE still?

Hah, I don't think I do, otherwise it would be listed in [browsers.json](https://github.com/code-dot-org/code-dot-org/blob/e3b376823e5c81b85f434dd5e87943fa3e1636a1/dashboard/test/ui/browsers.json).  Looks like we removed it [here](https://github.com/code-dot-org/code-dot-org/pull/44336).

Reply:
Cool! I see Darin wrote in the PR that removed IE:
```
Not removing IE references from
- test cases with @eyes_ie or @no_ie annotations
- assertions of IE compatability in "activity-guidelines" or other documents
We recommend that teams spend some time searching for any cleanup opportunities.
```
I'll go ahead and remove this tag - should I go ahead and remove this tag and comments/references to it in other files in this PR or in a separate PR?

Reply:
https://codedotorg.atlassian.net/browse/SL-786